server/admin/src/main/java/org/kaaproject/kaa/server/admin/controller/KaaAdminController.java

```diff
@@ -150,6 +171,90 @@ public void handleKaaAdminServiceException(KaaAdminServiceException ex, HttpServ
     }
 
     /**
+     * Gets the endpoint profile by endpoint group id.
+     */
+    @RequestMapping(value = "endpointProfileByGroupId", method = RequestMethod.GET, produces = APPLICATION_JSON_VALUE)
+    @ResponseBody
+    public EndpointProfilesPageDto getEndpointProfileByEndpointGroupId(
+            @RequestParam(value = "endpointGroupId") String endpointGroupId,
+            @RequestParam(value = "limit", defaultValue = DEFAULT_LIMIT) String limit,
+            @RequestParam(value = "offset", defaultValue = DEFAULT_OFFSET) String offset,
+            HttpServletRequest request) throws KaaAdminServiceException {
+        EndpointProfilesPageDto endpointProfilesPageDto = kaaAdminService.getEndpointProfileByEndpointGroupId(endpointGroupId, limit, offset);
+        if (endpointProfilesPageDto.hasEndpointProfiles()) {
+            PageLinkDto pageLinkDto = createNext(endpointProfilesPageDto.getPageLinkDto(), request);
+            endpointProfilesPageDto.setPageLinkDto(pageLinkDto);
+        }
+        return endpointProfilesPageDto;
+    }
+
+    /**
+     * Gets the endpoint profile body by endpoint group id.
+     */
+    @RequestMapping(value = "endpointProfileBodyByGroupId", method = RequestMethod.GET, produces = APPLICATION_JSON_VALUE)
+    @ResponseBody
+    public EndpointProfilesBodyDto getEndpointProfileBodyByEndpointGroupId(
+            @RequestParam(value = "endpointGroupId") String endpointGroupId,
+            @RequestParam(value = "limit", defaultValue = DEFAULT_LIMIT) String limit,
+            @RequestParam(value = "offset", defaultValue = DEFAULT_OFFSET) String offset,
+            HttpServletRequest request) throws KaaAdminServiceException {
+        EndpointProfilesPageDto endpointProfilesPageDto = kaaAdminService.getEndpointProfileBodyByEndpointGroupId(endpointGroupId, limit, offset);
```

Comment:
Why do we use different method and expect EndpointProfilesPageDto object, after that convert to another object?